Assets/UI/bevy_mod_bbcode.toml

```diff
@@ -0,0 +1,4 @@
+name = "bevy_mod_bbcode"
+description = "Format your UI text using BBCode!"
```

Comment:
```suggestion
description = "Format your UI text using the BBCode markup language!"
```

Reply:
Something like this would be clearer and easier to search IMO.

Reply:
Sounds good!

Im on my phone now and cant change it, feel free to apply the suggestion or Ill do it tomorrow :)